Header Header

line_polar far_left
line_polar left
line_polar right
line_polar far_right

# Although there are four lines in this message, some of them may be empty if they do not exist in the environment.
# It is also possible some lines are discarded because there is no room for them in the message
# It is the responsibility of the driving controller to handle these lines.